docs/source/storeconv.md

```diff
@@ -74,11 +78,11 @@ Every implementation of the `xcube.core.store.DataOpener` or
 allowed arguments to `open_data` for each dataset supported by the
 `DataOpener` or `DataStore`. The description is provided as a
 `JsonObjectSchema` object corresponding to a [JSON
-Schema](https://json-schema.org/). The intention is that this description be
-full and detailed enough to allow the automatic construction of a user
-interface for access to the available datasets. Note that, under this system:
+Schema](https://json-schema.org/). The intention is that this description should be 
+detailed enough to allow the automatic construction of a user
+interface for accessing the available datasets. Note that, under this system:
```

Comment:
I'm not convinced that the rephrasing is an improvement... but perhaps after all it's safest to avoid the subjunctive  in technical documents, so I won't object :).

Reply:
I agree with Pont.